src/scss/_buttons.scss

```diff
@@ -15,6 +15,12 @@ button{
         background: var(--theme-off-bg-color);
     }
 }
+.button-tip-label {
+    display: block; 
+    font-size: 10px;
+    color: darkgreen;
```

Comment:
You shouldn't use this color since it's not part of the theme, and is not *derived* from the theme colors either. For example in this case, you can use `color: rgba(var(--rgb-black), .5);`, or something along those lines, as defined in:

https://github.com/imolorhe/altair/blob/staging/src/scss/_variables.scss